src/app/component/home/components/eco-events/eco-events.component.ts

```diff
@@ -24,7 +24,7 @@ export class EcoEventsComponent implements OnInit {
     this.newsService.loadLatestNews();
     this.newsService.latestNews.pipe(catchError(() => of([]))).subscribe(
       (newsItems: NewsDto[]) => {
-        newsItems.forEach(
+        newsItems.map(
           (element: NewsDto) => (element.creationDate = this.convertDate(element.creationDate))
```

Comment:
Bad and sadly. It doesn't make sense to replace `forEach` on `map` a formal.  What’s more, you spoiled the functionality.
```js
someArray.forEach(el => el = newElValue);
```
is not equal to the
```js
someArray.map(el => el = newElValue);
```
- check it!

Reply:
Fixed